src/Plugin/Filter/FilterVisualContent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
@KevinVillanuevaCordero Please removed this comments and replace it with the comment @file